pkg/tsdb/schema/schema.go

```diff
@@ -0,0 +1,68 @@
+package schema
+
+import (
+	"github.com/pkg/errors"
+	"github.com/v3io/v3io-tsdb/pkg/aggregate"
+	"github.com/v3io/v3io-tsdb/pkg/config"
+	"strings"
+)
+
+const (
+	SchemaVersion                 = 0
+	DefaultStorageClass           = "local"
+	DefaultIngestionRate          = ""
```

Comment:
It is not used. either delete it for now, or calculate the chunk&partition interval from the rate.

Reply:
user MUST provide `IngestionRate` therefore NO DEFAULT value